website/docs/docs/collaborate/environments/dbt-cloud-environments.md

```diff
@@ -0,0 +1,222 @@
+
+### Setting developer credentials
+
+To use the IDE, each developer will need to set up [personal development credentials](/docs/get-started/develop-in-the-cloud#developer-credentials) to your warehouse connection in their `Profile Settings`. This allows users to set separate target information, as well as maintain individual credentials to connect to your warehouse via the dbt Cloud IDE.
+
+
+<Lightbox src="/img/docs/dbt-cloud/refresh-ide/new-environment-fields.png" width="100" height="100" title="Creating a development environment"/>
+
+## Create a deployment environment
+
+To create a new dbt Cloud development environment, navigate to the `Environments` page under the `Deploy` header menu, and click `Create Environment` . Select `Deployment` as the environemnt type.
+
+### Semantic Layer
+
+For Semantic Layer-eligible customers, the next section of environment settings is the Semantic Layer configurations. [The Semantic Layer setup guide](/docs/use-dbt-semantic-layer/setup-dbt-semantic-layer) has the most up-to-date setup instructions!
+
+### Deployment Connection
+
+:::info Warehouse Connections
+
+ Warehouse connections are set at the Project level for dbt Cloud accounts, and each Project can have one connection (Snowflake account, Redshift host, Bigquery project, Databricks host, etc.). Some details of that connection (databases/schemas/etc.) can be overridden within this section of the dbt Cloud environment settings.
+
+:::
+
+This section determines the exact location in your warehouse dbt should target when building warehouse objects! This section will look a bit different depending on your warehouse provider.
+
+<WHCode>
+
+
+<div warehouse="Postgres">
+
+This section will not appear if you are using Postgres, as all values are inferred from the project's connection.
+
+</div>
+
+<div warehouse="Redshift">
+
+This section will not appear if you are using Redshift, as all values are inferred from the project's connection.
+
+</div>
+
+<div warehouse="Snowflake">
+
+<Lightbox src="/img/docs/collaborate/snowflake-deploy-env-deploy-connection.png" title="Snowflake Deployment Connection Settings"/>
+
+#### Editable fields
+
+- `Role`: Snowflake Role
+- `Database`: Target Database
+- `Warehouse`: Snowflake Warehouse
+
+</div>
+
+<div warehouse="Bigquery">
+
+This section will not appear if you are using Bigquery, as all values are inferred from the project's connection.
+
+</div>
+
+<div warehouse="Spark">
+
+This section will not appear if you are using Spark, as all values are inferred from the project's connection.
+
+</div>
+
+<div warehouse="Databricks">
+
+<Lightbox src="/img/docs/collaborate/databricks-deploy-env-deploy-connection.png" title="Databricks Deployment Connection Settings"/>
+
+This section will not appear if you are using BigQuery, as all values are inferred from the project's connection. The `dataset` will be set in the `Deployment Credentials` section below.
```

Comment:
@dave-connors-3 it looks like the first sentence ('this section will not appear..') shouldn't be here since this is a databricks tab. is it fine to remove it?```suggestion
This section will not appear if you are using BigQuery, as all values are inferred from the project's connection. The **dataset** will be set in the [**Deployment Credentials**](#deployment-credentials) section below.
```

Reply:
![Screenshot 2023-02-07 at 10 37 14](https://user-images.githubusercontent.com/89008547/217221893-335ec51d-734d-4c6b-ab5b-f66d3236de31.png)

Reply:
yep, i will double check that!